recipe/meta.yaml

```diff
@@ -238,13 +236,6 @@ test:
     - test -f $PREFIX/share/cmake/Torch/TorchConfig.cmake                       # [linux]
     - if not exist %LIBRARY_PREFIX%\share\cmake\Torch\TorchConfig.cmake exit 1  # [win]
 
-    # test integrity of CMake metadata
-    - cd cmake_test
-    - cmake -GNinja -DCMAKE_CXX_STANDARD=17 $CMAKE_ARGS .   # [unix]
-    - cmake -GNinja -DCMAKE_CXX_STANDARD=17 %CMAKE_ARGS% .  # [win]
```

Comment:
Don't move the entire `cmake_test`. The config stage still needs to be tested in `libtorch`.